routers/web/org/times.go

```diff
@@ -0,0 +1,218 @@
+// Copyright 2014 The Gogs Authors. All rights reserved.
+// Copyright 2022 The Gitea Authors.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package org
+
+import (
+	"net/http"
+	"time"
+
+	"code.gitea.io/gitea/models/db"
+	"code.gitea.io/gitea/modules/base"
+	"code.gitea.io/gitea/modules/context"
+	"code.gitea.io/gitea/modules/setting"
+
+	"xorm.io/builder"
+)
+
+const (
+	tplByRepos      base.TplName = "org/times/times_by_repos"
+	tplByMembers    base.TplName = "org/times/times_by_members"
+	tplByMilestones base.TplName = "org/times/times_by_milestones"
+)
+
+// resultTimesByRepos is a struct for DB query results
+type resultTimesByRepos struct {
+	Name    string `xorm:"name"`
+	SumTime int64  `xorm:"sumtime"`
+}
+
+// resultTimesByMilestones is a struct for DB query results
+type resultTimesByMilestones struct {
+	RepoName     string `xorm:"reponame"`
+	Name         string `xorm:"name"`
+	ID           string `xorm:"id"`
+	SumTime      int64  `xorm:"sumtime"`
+	HideRepoName bool   `xorm:"-"`
+}
+
+// resultTimesByMembers is a struct for DB query results
+type resultTimesByMembers struct {
+	Name    string `xorm:"name"`
+	SumTime int64  `xorm:"sumtime"`
```

Comment:
Usually you only use `xorm` tags for struct that will be stored into the database. Xorm should be able to figure out by itself which row corresponds to a field(if not, then add a xorm tag).

Reply:
Changed. To let Xorm figure out by itself, the query builder part was also touched (`sumtime` => `sum_time`, so it recognizes `SumTime`, etc.